Classes/Fluid/ViewHelper/ComponentRenderer.php

```diff
@@ -519,4 +523,16 @@ protected function getComponentArgumentConverter(): ComponentArgumentConverter
     {
         return GeneralUtility::makeInstance(ComponentArgumentConverter::class);
     }
+
+    /**
+     * @return bool
+     */
+    protected static function shouldUseTemplatePaths(): bool
+    {
+        static $assertion = null;
+        if ($assertion === null) {
+            $assertion = GeneralUtility::makeInstance(Features::class)->isFeatureEnabled('FluidComponentsTemplatePaths');
```

Comment:
In case the suggestion works, we should use `fluidComponents.partialsInComponents` here because this describes the feature better. Also, it conforms to the naming convention for feature flags (`lowerCaseCategory.lowerCaseFeature`):
https://github.com/TYPO3/TYPO3.CMS/blob/master/typo3/sysext/core/Configuration/DefaultConfigurationDescription.yaml#L211